src/features/game/scenes/chapterSelect/ChapterSelectHelper.ts

```diff
@@ -152,8 +156,19 @@ export function createChapter(
   return chapterContainer;
 }
 
-export function getCoorByChapter(chapterNum: number) {
-  const x = screenCenter.x + chapConstants.imageDist * chapterNum;
-  const y = screenCenter.y;
+function getCoorByChapter(chapterNum: number) {
+  const page = Math.floor(chapterNum / chapConstants.grid.chapPerPage);
+  chapterNum %= chapConstants.grid.chapPerPage;
+  const row = Math.floor(chapterNum / chapConstants.grid.chapPerRow);
+  const col = chapterNum % chapConstants.grid.chapPerRow;
+
+  const centreCol = chapConstants.grid.chapPerRow / 2 - 0.5;
```

Comment:
The 0.5 could seem a tad cryptic. Maybe `(row - 1) / 2` would be slightly clearer